--- cfitsio/configure.ori	2018-10-02 09:15:43.000000000 -0700
+++ cfitsio/configure	2018-10-02 09:15:54.000000000 -0700
@@ -3904,7 +3904,7 @@
                 # Test to see whether the C compiler accepts the "-arch"
                 # flags for building "universal" binaries (Apple XCode only):
                 SAVE_CFLAGS="$CFLAGS"
-                C_UNIV_SWITCH="-arch i386 -arch x86_64"
+                C_UNIV_SWITCH="-arch x86_64"
                 CFLAGS="$CFLAGS $C_UNIV_SWITCH"
                 { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether $CC accepts $C_UNIV_SWITCH" >&5
 $as_echo_n "checking whether $CC accepts $C_UNIV_SWITCH... " >&6; }
